Scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,439 @@
+@prefix scenario: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/scenario/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+scenario:scenarioShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:EnvironmentalShape ;
+            sh:order 2 ;
+            sh:path scenario:Environmental ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:GeneralShape ;
```

Comment:
There is also a "general:GeneralShape" defined by Mirco (#27)
This one seems to be something different. Is "General" the right name for it?

As I understood Mirco all DataShapes (like HdMap) should include the general:GeneralShape. Did you forget it to include?

Reply:
Yes, this is scenario-related general content. As this definitions are originated from SRMD / SETlevel-semantics, I would propose to keep it here

Reply:
To prevent ambiguities, we could rename it to ScenarioGeneralShape. Would this help?

Reply:
As aligned, will be renamed